.github/workflows/ci-workflow.yml

```diff
@@ -79,11 +79,12 @@ concurrency:
   cancel-in-progress: true
 
 permissions:
+  security-events: write
```

Comment:
Be interested to see if you get this to work, I tried to improve the test result reporting, but ran into apparent write permission issues, so raised [NIFI-12192](https://issues.apache.org/jira/browse/NIFI-12192)

Might not be a permissions issue though or could be a different reason that this appears to have failed so far (I didn't spend very long looking into the issue previously, and it was using a different action)